Doc/c-api/init_config.rst

```diff
@@ -828,6 +828,18 @@ PyConfig
 
       Default: ``0``.
 
+   .. c:member:: int32_t int_max_str_digits
+
+      If greater than 0, enable int conversion digit limitations. ``-1`` means
```

Comment:
You can add a link on "int conversion digit limitations"? I recall that there is a dedicated section in the doc.